jobs/bbr-credhubdb/spec

```diff
@@ -39,3 +39,6 @@ properties:
     default: true
   credhub.data_storage.tls_ca:
     description: "CA trusted for making TLS connections to targeted database server"
+  credhub.data_storage.hostname_verification.enabled:
+    description: "Toggle certificate host name verification. WARNING: Only use if your database does not include a matching hostname in their server certificate!"
```

Comment:
I like that you added the warning. Maybe could it be a bit more precise. Something like:
If your database server certificate contains the correct hostname, the value `true` is strongly recommended, as disabling hostname verification results in lesser security.

Reply:
I have updated the warning to  reflect your comment

Reply:
Great. I would appreciate it if you could quote the "true" keyword in the sentence please, as I find that it makes the sentence more readable. 
> the value true

may be a bit confusing to maybe non-native english speakers. While

> the value "true"

should be clear to everyone.

Reply:
I actually don't know how much of a pain it is to escape quotes inside of the description string. I usually strongly prefer double quotes (") in error messages for quoting, but here single quotes/apostrophes (') may be the way to go.

Reply:
I added single quotes around the word "true" because I suspect double quotes will break it with bosh. It breaks when parsing the file with yq